digdag-spi/src/main/java/io/digdag/spi/CommandStatus.java

```diff
@@ -13,11 +14,20 @@
     boolean isFinished();
 
     /**
-     * Return exit code of command finished. It7s valid only when isFiished returns true.
+     * Return exit code of command finished. It is valid only when isFinished returns true.
```

Comment:
[just an idea] Maybe we'd better merge `isFinished` and `statusCode` into an `Optional<Integer>` in the future so that we can avoid unexpectedly using status code when not finished.

Reply:
Make sense for me. I will add **ToDo** about it.